scripts/exregional_pre_post_stat.sh

```diff
@@ -53,10 +53,22 @@ This is the ex-script for the task that runs POST-UPP-STAT.
 #
 eval ${PRE_TASK_CMDS}
 
-if [ ${#FCST_LEN_CYCL[@]} -gt 1 ]; then
-  cyc_mod=$(( ${cyc} - ${DATE_FIRST_CYCL:8:2} ))
-  CYCLE_IDX=$(( ${cyc_mod} / ${INCR_CYCL_FREQ} ))
-  FCST_LEN_HRS=${FCST_LEN_CYCL[$CYCLE_IDX]}
+if [ "${FCST_LEN_HRS}" = "-1" ]; then
```

Comment:
Same as above

Reply:
Same explanation as above. Recommended to keep.